packages/hardhat/contracts/strategy/CrabStrategy.sol

```diff
@@ -578,6 +577,7 @@ contract CrabStrategy is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Ownab
         bool _isStrategySellingWSqueeth,
         uint256 _limitPrice
     ) internal {
+        require( _auctionTriggerTime > timeAtLastHedge, "Trigger time must be after time at last hedge");
```

Comment:
This require is partially checked anytime we are doing a timeHedge via _isTimeHedge(): 
`       
 uint256 auctionTriggerTime = timeAtLastHedge.add(hedgeTimeThreshold);
`

So I think it might be cheaper/better to only check this         
`
require( _auctionTriggerTime > timeAtLastHedge, "Trigger time must be after time at last hedge");
`
in the isPriceHedge() check.  Thoughts?

Reply:
yeah good idea